### PR TITLE
Webex Bronze - App docs

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to create Webex Bronze apps docs.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Webex App - Create Dochub page](https://sumologic.atlassian.net/browse/CONN-3393)

### CODE REVIEW

File: blog-service/2024-07-19-app.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - apps
+  - app-catalog
+  - webex-bronze
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce the release of the new Webex Bronze app for Sumo Logic. This app leverages Sumo Logic [Cloud-to-Cloud Webex source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/webex-source) to provide comprehensive monitoring and analysis of Webex environments, offering real-time visibility into security events, user activities, and system health.
```
Comment (jpipkin1) [inline blog-service/2024-07-19-app.md:14]:
```suggestion
We're excited to announce the release of the new Webex Bronze app for Sumo Logic. This app leverages Sumo Logic [Cloud-to-Cloud Webex Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/webex-source) to provide comprehensive monitoring and analysis of Webex environments, offering real-time visibility into security events, user activities, and system health.
```

File: docs/integrations/saas-cloud/webex-bronze.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/webex-logo.png')} alt="webex-logo" width="120" />
+
+The Sumo Logic app for Webex provides comprehensive monitoring and analysis of Webex environments, offering real-time visibility into security events, user activities, and system health. This app includes pre-built dashboards and queries that help organizations quickly identify and respond to security incidents, track user behaviors, manage device activities, and ensure compliance. With detailed insights into login events, authorization activities, device management, and administrative actions, this app enhances operational efficiency and strengthens security postures by providing actionable intelligence.
+
+## Log type
+
+This app uses Sumo Logic’s [Webex Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/webex-source/) to collect [Admin Audit events](https://developer.webex.com/docs/api/v1/admin-audit-events/list-admin-audit-events) from Webex platform.
```
Comment (jpipkin1) [inline docs/integrations/saas-cloud/webex-bronze.md:16]:
```suggestion
This app uses Sumo Logic’s [Webex Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/webex-source/) to collect [Admin Audit events](https://developer.webex.com/docs/api/v1/admin-audit-events/list-admin-audit-events) from the Webex platform.
```

**Review by jpipkin1 — APPROVED** [review]
It looks great, @JV0812! I have only a few minor suggestions.

File: docs/integrations/saas-cloud/webex-bronze.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+// global filters
+| where event_category matches "{{event_category}}"
+| where event_description matches "{{event_description}}"
+
+| count
+```
+
+## Set up collection
+
+To set up [Webex Cloud-to-Cloud Integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/webex-source/) for the Webex Bronze app, follow the instructions provided. These instructions will guide you through the process of creating a source using the Webex Bronze Source category, which you will need to use when installing the app. By following these steps, you can ensure that your Webex Bronze app is properly integrated and configured to collect and analyze your Webex Bronze data.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/webex-bronze.md:83]:
```suggestion
To set up the Webex Cloud-to-Cloud Integration for the Webex Bronze app, follow the instructions provided [here](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/webex-source/). These instructions will guide you through the process of creating a source using the Webex Bronze Source category, which you will need to use when installing the app. By following these steps, you can ensure that your Webex Bronze app is properly integrated and configured to collect and analyze your Webex Bronze data.
```

**Comment by JV0812** [comment]
@jpipkin1 Thanks for your review and suggestion. I have fixed the comments.

**Review by kimsauce — APPROVED** [review]

**Comment by ankitgoelcmu** [comment]
LGTM

**Review by ankitgoelcmu — COMMENTED** [review]
LGTM

**Pull request merged**